var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -76,6 +79,17 @@ class Mumps(Package):
     conflicts('+blr_mt', when='~openmp',
               msg="You cannot use the blr_mt variant without openmp")
 
+    @when("+incfort")
+    def patch(self):
+        """Prevents type mismatch if the headers are used with '-i8'/'-r8'"""
```

Comment:
1. I think it is more frequent _not_ to conjugate the  verb in a docstring :   
`"""Prevent type mismatch if the headers are used with '-i8'/'-r8'"""`
3. The rationale in the PR is more clear to me than this docstring, I think you could add it in the docstring or in a comment
> This is very interesting in this usecase: compiling mumps with short int and building a program that includes these headers with -fdefault-integer-8/-i8

Reply:
That's done